apps/fishing-map/features/map/popups/PopupWrapper.tsx

```diff
@@ -83,9 +85,9 @@ function PopupWrapper({ interaction, type = 'hover', className = '', onClose }:
       }),
     ],
   })
-
   if (!mapViewport || !interaction || !interaction.features?.length) return null
-
+  // Map tools overlays are handled by each tool
+  if (isMapToolsActive) return null
```

Comment:
I think this fits better in the [MapPopups component](https://github.com/GlobalFishingWatch/frontend/blob/deck-migration/draw-layer-leftovers/apps/fishing-map/features/map/popups/MapPopups.tsx#L18)